src/site/pages/blog/article/Article.module.css

```
@@ -0,0 +1,98 @@
+}
+
+.articleContent {
+	--octc-color-bg: transparent;
+	--octc-color-bg-alt: #8881;
+	--octc-color-border: #8884;
+	--octc-color-primary: currentColor;
+	--octc-color-text: currentColor;
+	--octc-color-text-muted: #808080;
+	--octc-focus-offset: 2px;
+	--octc-focus-ring: 2px solid currentColor;
+	word-break: keep-all;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)